### DISCUSSION TITLE
How to use concat and offset plugins in combination

### DISCUSSION THREAD

**Discussion by kokaprathyush** [videojs/video.js#7568] [Q&A]
Hello, I am using concat plugin to concatenate multiple hls videos and play in sequence but is there a possibility to add start and end offsets to those videos.

For example, if there are two videos, I want the first video to play for 5 seconds and switch to the second video and once it is done  I want the first video to continue playing from the 5th second onwards. Is there any way to achieve this.

Thanks

**Comment by gkatsev** [comment]
The easiest is probably to use the offset plugin and a playlist plugin, and then re-init the offset for each video.

Using the concat plugin, you'll probably want to use the callback to filter out segments you don't want, while adding the appropriate discontinuity flags as needed. Though, this may be tricky if you have multiple playlists or demuxed audio.
